tests/product/test_topology.py

```diff
@@ -58,26 +58,6 @@ def test_topology_show(self):
         expected = normal_topology
         self.assertEqual(expected, actual)
 
-    @docker_only
```

Comment:
Can we unit test this? I assume it's just reading the config.json locally and returning that.